app/index.html

```diff
@@ -21,12 +21,6 @@
     {{content-for "head-footer"}}
   </head>
   <body class="responsive status">
-    <!--[if lt IE 9]>
-    <script type="text/javascript" src="https://emberjs.com/javascripts/common-old-ie-f9a6a6d5.js"></script>
-    <![endif]-->
-    <!--[if gte IE 9]><!-->
-    <script type="text/javascript" src="https://emberjs.com/javascripts/common-modern-8dbdc765.js"></script>
-    <!--<![endif]-->
```

Comment:
Can you remove the two script tags at the bottom, too?

Reply:
Yes, a second

Reply:
Oops, we can only do that once we use ember-styleguide going.

Reply:
This issue already exists #24